### PR TITLE
fix blues bt profile build issue

### PR DESCRIPTION
**Pull request by stevenatom**
Signed-off-by: Steven Chen <steven.chen@unisoc.com>

### CODE REVIEW

File: repeater/src/bluetooth/mesh.c [hunk 1]
```diff
@@ -274,7 +270,7 @@ void vendor_net_send(void)
 
 	err = bt_mesh_model_send(&vnd_models[0], &ctx, &msg, NULL, NULL);
 	if (err) {
-		LOG_ERR("Failed to send (err %d)", err);
+		SYS_LOG_ERR("Failed to send (err %d)", err);
```
Comment (easonxiang) [inline repeater/src/bluetooth/mesh.c]: We are using the new log subsystem, details:
https://docs.zephyrproject.org/latest/subsystems/logging/logger.html

File: repeater/src/bluetooth/blues.c [hunk 2]
```diff
@@ -25,30 +25,18 @@
...
 
-#include "../../../../drivers/bluetooth/unisoc/uki_utlis.h"
-#include "../../../../drivers/bluetooth/unisoc/uki_config.h"
+#include <uwp5661/drivers/src/bt/uki_utlis.h>
+#include <uwp5661/drivers/src/bt/uki_config.h>
 #include "blues.h"
 #include "throughput.h"
 #include "wifi_manager_service.h"
 #include "mesh.h"
+#include "light.h"
```
Comment (easonxiang) [inline repeater/src/bluetooth/blues.c]: We have implement the led driver in zephyr/drivers/led/uwp.c, the usage can be found in function cmd_led of apps/repeater/src/test.c.

Comment (bublover) [reply repeater/src/bluetooth/blues.c]: It's a simple LED driver, only turn on / off  PIN1 and PIN3 for UWP_GPIO_P0.
You might add something you want.

File: repeater/src/bluetooth/mesh.c (hunk 1 above)
Comment (stevenatom) [reply repeater/src/bluetooth/mesh.c]: got it

**Pull request merged**